### PR TITLE
Enable different cache eviction policy

### PR DESCRIPTION
**Pull request by apc999**
Test plan - Unit test added

### CODE REVIEW

**Review by jainxrohit — APPROVED** [review]

File: presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioCacheConfig.java [hunk 1]
```diff
@@ -33,7 +33,8 @@
     private boolean cacheQuotaEnabled;
     private Duration timeoutDuration = new Duration(60, SECONDS);
     private int timeoutThreads = 64;
-    private int evictionRetries;
+    private int evictionRetries = 10;
+    private String evictionPolicy = "alluxio.client.file.cache.evictor.LRUCacheEvictor";
```
Comment (kewang1024) [inline presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioCacheConfig.java:37]: On the service level, we shouldn't use this long string as the config which has too much implementing detail, we should change it to something simpler and straightforward
If we can do it in this PR, we should do it
If we can't, we should do it in the next release

Comment (shixuan-fan) [reply presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioCacheConfig.java:37]: Agree with @kewang1024 , we should use an enum. A random string is very error-prone for a policy.

**Review by shixuan-fan — CHANGES_REQUESTED** [review]

File: presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioCacheConfig.java (hunk 1 above)
Comment (apc999) [reply presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioCacheConfig.java:37]: @kewang1024 @shixuan-fan  do you have an example of using enum as config in Presto code base?

Comment (kewang1024) [reply presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioCacheConfig.java:37]: https://github.com/prestodb/presto/blob/a2d55c1deb74f2ababde29da804d235d44dccda8/presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java#L252-L256

**Comment by apc999** [comment]
@shixuan-fan @kewang1024  PTAL

**Comment by apc999** [comment]
@shixuan-fan @kewang1024 ping

**Review by shixuan-fan — APPROVED** [review]
LGTM

**Pull request merged**